Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/download/CheckInsPackage.kt

```diff
@@ -27,41 +28,45 @@ private const val TYPE_EVENT = 2
 private val dummyEventCheckIn1: CheckIn = CheckIn(
     id = 1L,
     guid = "eventOne",
+    guidHash = ByteArray(0),
     version = 1,
     type = TYPE_LOCATION,
     description = "Restaurant",
     address = "Around the corner",
     traceLocationStart = null,
     traceLocationEnd = null,
     defaultCheckInLengthInMinutes = null,
-    signature = "signature",
+    traceLocationBytes = ByteArray(0),
+    signature = "c2lnbmF0dXJl".decodeBase64()!!,
```

Comment:
Use `"signature".toByteArray().toByteString()` instead?

Reply:
It doesn't matter, I'm changing this anyway as the download has a different data structure

Reply:
ok